### ISSUE TITLE
Add react redbox integration with react hot loader

### ISSUE THREAD

**Issue by michaelBenin** [michaelBenin/react-ssr-spa#7]
https://github.com/commissure/redbox-react

Example with react-hot-loader:

https://github.com/commissure/redbox-react/blob/master/examples/react-hot-loader/index.js

**Comment by michaelBenin** [comment]
Waiting on this pr: https://github.com/facebook/react/pull/6020

**Comment by michaelBenin** [comment]
No longer needed with ErrorBoundary. #66
